### PR TITLE
Compatibility: Gravity Forms + Yoast fix

### PR DESCRIPTION
**Pull request by AlexGStapleton**
Resolve https://github.com/siteorigin/siteorigin-panels/issues/321

This PR will run [gform_disable_print_form_scripts](https://docs.gravityforms.com/gform_disable_print_form_scripts/) when the Gravity Forms widget is added to a Page Builder layout.

This PR adds `siteorigin_panels_widget_instance` ([docs PR](https://github.com/siteorigin/docs/pull/105))

### CODE REVIEW

**Comment by Misplon** [comment]
Thanks. The issue seems to be persisting for me. Here is what I see:

Edit_Page_‹_SiteOrigin_—_WordPress

**Comment by AlexGStapleton** [comment]
The provided fix won't adjust the setting as that's overly complicated and potentially confusing ("why does this setting keep getting re-enabled?"). Instead, it overrides the setting on the fly using a filter.

Can you please [send me a copy of the form](https://docs.gravityforms.com/exporting-a-form-from-gravity-forms/) you're testing? To be clear, this issue is only visible when using the form, and not while editing it.

**Comment by Misplon** [comment]
> The provided fix won't adjust the setting as that's overly complicated and potentially confusing ("why does this setting keep getting re-enabled?"). Instead, it overrides the setting on the fly using a filter.

Thanks, I'm not expecting this PR to adjust the setting, my screenshot is just showing everything I see.

> To be clear, this issue is only visible when using the form, and not while editing it.

Using the form I'm testing with, there isn't an error on the site front-end using the develop branch. Does the form need conditional fields?

When enabling the Disable script output setting, the JS error does resolve in the admin. Should we not also use the filter in the admin to prevent the error?

**Comment by Misplon** [comment]
Test form: 
[gravityforms-export-2020-08-25.json.zip](https://github.com/siteorigin/siteorigin-panels/files/5122474/gravityforms-export-2020-08-25.json.zip)

**Comment by AlexGStapleton** [comment]
I'm not getting the error message you're getting when editing the widget. I'm also having trouble replicating this error again. 

> Does the form need conditional fields?

Yes. [Here's the form](https://drive.google.com/uc?id=1f48BtNFq6KRvvSM7DClvcmIh8TQKS3TH) I was using when I initially created this PR. Mind you, I'm having trouble replicating it now. :/
I hate to say it, but it might be worth closing this and suggesting disabling "disable script output" if it happens. It feels safer.

**Comment by Misplon** [comment]
Thanks, I've tried your form and I've tried adding a conditional field to my first test form, I'm now not able to recreate a console error in the admin or front-end.

File: compat/gravity-forms.php [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<?php
+/**
+ * Disable Gravity Disable Print Form Scirpts.
```
Comment (Misplon) [inline compat/gravity-forms.php:3]: Gravity Forms Disable Print Scripts?

**Review by Misplon — CHANGES_REQUESTED** [review]
Description update/typo correction required, otherwise all good.

**Review by Misplon — APPROVED** [review]

**Comment by Misplon** [comment]
@AlexGStapleton Can you take a look at the above conflict? Thanks, mate.

**Comment by AlexGStapleton** [comment]
Done.

**Pull request merged**